css-animationworklet/Overview.bs

```diff
@@ -456,6 +551,9 @@ To <dfn>migrate an animator instance</dfn> from one {{WorkletGlobalScope}} to an
 
 </div>
 
+Note: If an animator state getter throws the user agent will remove the animator but does not
```

Comment:
Is this normative text? If so, perhaps it shouldn't be a note?

Reply:
This was to make it clear that this is effectively the implication of the algorithm above. I remove the note to make it clear it is normative.